examples/custom_object3.py

```diff
@@ -86,60 +141,17 @@ def vertex_shader(self):
         }
         """
 
-    def fragment_shader(self):
+    def code_fragment(self):
         return """
         @stage(fragment)
         fn fs_main(varyings: Varyings) -> FragmentOutput {
             var out: FragmentOutput;
-            let a = u_material.opacity * u_material.color.a;
-            out.color = vec4<f32>(u_material.color.rgb, 1.0);
+            out.color = vec4<f32>(u_material.color.rgb, u_material.opacity);
```

Comment:
the old code looks wrong, and the new code doesn't use the colormap's alpha attribute. is that intended?

Reply:
mmm, I tried to simplify the render_mask logic by not using the color's alpha, but this is confusing in other ways. I made it more like we do in the real materials: multiply opacity with the color's alpha.